app/1.3.x/proxy.md

```diff
@@ -761,7 +761,16 @@ allows routing via SNIs:
 Incoming requests with a matching hostname set in the TLS connection's SNI
 extension would be routed to this Route. As mentioned, SNI routing applies not
 only to TLS, but also to other protocols carried over TLS - such as HTTPS and
-gRPCs.
+gRPCs. If more than one hostnames are specified, then any one of them can match
```

Comment:
```suggestion
If multiple SNIs are specified in the Route, any of them can match with the incoming request's SNI.
```

Reply:
Note sure what we meant with "(OR relationship between the names)."; it sounds a little obscure to me.